tests/Unit/DataStructures/Test_VectorImpl.cpp

```diff
@@ -37,24 +37,3 @@ static_assert(not is_derived_of_vector_impl_v<std::complex<double>>,
 static_assert(not is_derived_of_vector_impl_v<std::vector<int>>,
               "Failed testing type trait is_derived_of_vector_impl");
 
-// [[OutputRegex, Comparing an empty vector to a value]]
-[[noreturn]] SPECTRE_TEST_CASE(
-    "Unit.DataStructures.VectorImpl.empty_equals_double",
-    "[DataStructures][Unit]") {
-  ASSERTION_TEST();
-#ifdef SPECTRE_DEBUG
-  (void)(DataVector{} == 1.0);
-  ERROR("Failed to trigger ASSERT in an assertion test");
-#endif
-}
-
-// [[OutputRegex, Comparing an empty vector to a value]]
-[[noreturn]] SPECTRE_TEST_CASE(
-    "Unit.DataStructures.VectorImpl.empty_equals_complex",
-    "[DataStructures][Unit]") {
-  ASSERTION_TEST();
-#ifdef SPECTRE_DEBUG
-  (void)(ComplexDataVector{} != std::complex<double>{});
-  ERROR("Failed to trigger ASSERT in an assertion test");
-#endif
-}
```

Comment:
This deletion seems to be in the wrong commit, so please correct if you get the chance, but no reason to hold up the PR.

Reply:
I believe this tested a piece of functionality that is now delegated to blaze, so no spectre assert is triggered for this error condition, so I think it makes sense in the 'remove PointerVector' commit.